apps/fluent-tester/src/E2E/Pressable/specs/Pressable.spec.win.ts

```diff
@@ -10,6 +10,10 @@ describe('Pressable Testing Initialization', function () {
   });
 
   it('Click and navigate to Pressable test page', async () => {
+    /* Scroll to component test page button in scrollview if not already visible*/
+
+    await PressablePageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
```

Comment:
The win32 E2E tests are failing from Pressable onward. I think this is because the buttons for the test pages in the test list are not in view. Do we need this call for `waitForButtonDisplayed`? Or can we just call `clickAndGoToPressablePage()` so that it automatically scrolls and clicks on the button for the test page?

Reply:
+1 `await PressablePageObject.waitForButtonDisplayed(PAGE_TIMEOUT);` is not needed here (or anywhere else in win32/macos specs). It will always fail